### PR TITLE
#593 Sidenav scroll fix

### PR DESCRIPTION
**Pull request by Louisreed**
**Issue**

- Link:
- Summary:

**How**

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: hw_diag/templates/template_hyper.html [hunk 1]
```diff
@@ -240,34 +239,66 @@
...
+                
+                // Function to check sidenav size and set margin-left property accordingly
                 function checkSidenavSize() {
                   var sidenavSize = $('html').attr('data-sidenav-size');
-                
+              
                   if (sidenavSize === 'full') {
                     $('#navbar-toggler').css('margin-left', '0');
-                    console.log("sidenav closed");
+                    //console.log("sidenav closed");
```
Comment (shawaj) [inline hw_diag/templates/template_hyper.html:266]: Why are we leaving commented code in?

Comment (Louisreed) [reply hw_diag/templates/template_hyper.html:266]: I don't have that on my copy. Either way it should resolve it when this is reviewed.

**Comment by Louisreed** [comment]
@shawaj Should be good to go

**Comment by shawaj** [comment]
Also we need to update MANIFEST.in with the new locations of the fonts and the removed stylesheets

**Comment by Louisreed** [comment]
@shawaj Done

**Review by shawaj — APPROVED** [review]

**Pull request merged**